### ISSUE TITLE
install: reorder methods and add badges

### ISSUE THREAD

**Issue by casperdcl** [iterative/dvc.org#904]
Update options including order and badges.

- conda [![Conda-forge](https://anaconda.org/conda-forge/dvc/badges/version.svg)](https://anaconda.org/conda-forge/dvc) or [![Conda-forge](https://img.shields.io/conda/v/conda-forge/dvc.svg?label=conda-forge&logo=conda-forge)](https://anaconda.org/conda-forge/dvc)
- snap [![Snapcraft](https://img.shields.io/badge/snap-install-82BEA0.svg?logo=snapcraft)](https://snapcraft.io/dvc) or [![dvc](https://snapcraft.io//dvc/badge.svg)](https://snapcraft.io/dvc)
  + also [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/dvc) or [![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/dvc)
- pip [![PyPI-Status](https://img.shields.io/pypi/v/dvc.svg?logo=PyPI&logoColor=white)](https://pypi.org/project/dvc)
- GitHub donwloads [![GitHub release (latest by date)](https://img.shields.io/github/v/release/iterative/dvc?label=GitHub&logo=github)](https://github.com/iterative/dvc/releases/latest)
  + deb
  + pkg
  + rpm
  + exe
- apt (?)
- choco [![Chocolatey Version](https://img.shields.io/chocolatey/v/dvc)](https://chocolatey.org/packages/dvc)
- brew [![homebrew](https://img.shields.io/homebrew/v/dvc)](https://formulae.brew.sh/formula/dvc)
- anything else?

Probably need to split first based on OS:

- Linux
  + conda
  + snap
  + deb/rpm
  + pip
- Mac
  + conda
  + brew
  + pip
  + pkg
- Windows
  + conda
  + choco
  + pip
  + exe

**Comment by jorgeorpinel** [comment]
Sounds like it's mostly about Windows @casperdcl. Would you like to open a PR directly for this? I think you worked on the `snap` method, for example.

**Comment by casperdcl** [comment]
I didn't intend it to be only about Win. In my experience there tends to be different ordering & options for Linux/Win/Mac. Could make a PR but probably needs discussion first from users of different OSs how they prefer/recommend installing

**Comment by jorgeorpinel** [comment]
> probably needs discussion first

It may be worth it to review past issues related to this, as we recently completely rewrote the installation section. e.g. #701/#734, #660/#701/#734, #656/695, others?

> else include "binary" pyinstaller packages 

Related: #65 (That's an old one!)

**Comment by casperdcl** [comment]
I think there are 2 major categories to split up: OS and end-user proficiency. Casual users can't be expected to go through the lengthy "separate conda env with dvc script appended to global `$PATH`" I posted elsewhere. Actually I'd say casual users should stick to snap/choco/brew. Advanced users meanwhile probably don't need much guidance - they just need to be aware what packages exist.

**Comment by shcheklein** [comment]
@casperdcl sounds good.

> Advanced users meanwhile probably don't need much guidance

It seems an obvious though, but my recent experience makes me think differently a bit. And probably, the reason for this that being even an advanced CLI user does not make you immediately an expert in different ways to install things or different environments. 

Probably they do not fit then into your meaning of advanced though, right? :) and should stick with some default choice.

**Comment by casperdcl** [comment]
> Probably they do not fit then into your meaning of advanced though, right? :) and should stick with some default choice.

Indeed. In this context I was using "advanced" to refer to knowledge of their preferred installation mechanism.

**Comment by jorgeorpinel** [comment]
Shall we include `snap` here too and close #918? Cc @casperdcl please notice that other issue's desc:

> as we discussed, could you please add this method to the docs as snap is official now?

**Comment by casperdcl** [comment]
for the record (vis https://github.com/iterative/dvc.org/issues/918#issuecomment-575936843) this issue is about documenting a new option while #918 is about reordering options.

**Comment by jorgeorpinel** [comment]
I think you meant this one is about reordering and #918 about a new option @casperdcl.

**Comment by fabiosantoscode** [comment]
Don't forget brew works on linux!

**Comment by casperdcl** [comment]
also potentially badges (vis https://github.com/iterative/dvc.org/issues/918#issuecomment-577345881)

**Comment by jorgeorpinel** [comment]
> **Windows**:
> conda
> choco
> pip
> exe

Why is conda on top? choco is probably the easiest from cmd line for most cases, no?

**Comment by shcheklein** [comment]
@jorgeorpinel it's a good question. I don't know how popular choco is to be honest ... it's as popular as brew for example?

**Comment by casperdcl** [comment]
I think maybe `conda` was on top for consistency across all platforms but I agree we should probably change to `conda` as second for all.

**Comment by jorgeorpinel** [comment]
> it's as popular as brew

My impression is that it's the equivalent to brew, yes. But much fewer people work on command line in Windows just because. Probably more people use Anaconda Prompt just because it's such a popular DS Python disto. But then again Git Bash is probably also quite popular (comes included in a bunch of other tools like Cmder or Git for Windows), and choco would be a better fit in those cases... so yeah not sure.

For now I put choco on top in https://github.com/iterative/dvc.org/pull/1195

**Comment by jorgeorpinel** [comment]
I believe we did take care of this from seeing https://dvc.org/doc/install

**Comment by jorgeorpinel** [comment]
p.s. well except for the badges but do we want those in docs? Maybe in the README

**Comment by jorgeorpinel** [comment]
p.p.s. and the badges do [exist in the README](https://github.com/iterative/dvc#installation) now. But the order there is not the same as in docs. Do you think we should reopen this and move it to the core repo @casperdcl ?

**Comment by casperdcl** [comment]
probably could be moved to core - would be a first iteration on updating somewhat neglected readmes.

https://github.com/iterative/terraform-provider-iterative/#readme just got an overhaul (https://github.com/iterative/terraform-provider-iterative/pull/433) for reference/review :)
